_config.yml

```diff
@@ -57,6 +55,13 @@ exclude:
 
 gems:
   - jekyll-redirect-from
+  - jekyll-multiple-languages-plugin
+
+help_pages:
```

Comment:
Sorry I meant this bit with English names that are used for navigation, like I'd expect it to be in an `_i18n/*.yml` not here anymore since it's got English text